src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -240,7 +240,10 @@ const SubmissionDetails = _.flow(
               size: { basis: 200 },
               headerRenderer: () => h(Sortable, { sort, field: 'messages', onSort: setSort }, ['Messages']),
               cellRenderer: ({ rowIndex }) => {
-                return h(TooltipCell, [h(Fragment, _.map(div, filteredWorkflows[rowIndex].messages))])
+                const messages = _.join('\n', filteredWorkflows[rowIndex].messages)
+                return h(TooltipCell, {
+                  tooltip: div({ style: { whiteSpace: 'pre-wrap', overflowWrap: 'break-word' } }, [messages])
```

Comment:
Is it possible to copy the contents of the tooltip? That's often the first thing I want to do when I have an error.

Reply:
You can triple-click on the cell to highlight the full text and then copy it, though that's not at all obvious if you don't already know about it.